next.clayui.com/content/docs/components/buttons.mdx

```diff
@@ -0,0 +1,86 @@
+---
+title: "Buttons"
+description: "Buttons communicate an action to happen on user interaction."
+packageNpm: "@clayui/button"
+packageVersion: "3.0.0"
+packageStatus: "Beta"
+---
+
+import {ButtonBlock, ButtonDisplayTypes, ButtonGroup, ButtonMonospaced, ButtonWithStar, ButtonSmall} from '../../../src/components/clay/Button';
+
+<div class="nav-toc">
+
+- [Variants and Best Pratices](#variants-and-best-pratices)
```

Comment:
I'm not sure if we want to add all the examples and use cases in the documentation, we're half describing again what the API is saying below. Maybe adding just one example of the button and one of the group is enough.

Reply:
I agree, I think the API should describe it enough and then we can also have one or two basic examples for demo purposes. I don't think we need examples of every variation and best practice though.